src/main/java/org/alliancegenome/curation_api/model/entities/AnatomicalSite.java

```diff
@@ -53,6 +55,19 @@ public class AnatomicalSite extends AuditedObject {
 	@ManyToOne
 	@JsonView({View.FieldsOnly.class})
 	private GOTerm cellularComponentTerm;
+	
+	//celullar compoent ribbon -- slim
+	@IndexedEmbedded(includePaths = {"name", "name_keyword"})
+	@IndexingDependency(reindexOnUpdate = ReindexOnUpdate.SHALLOW)
+	@ManyToOne
+	@JsonView({View.FieldsOnly.class})
+	private GOTerm cellularComponentRibbonTerm;
```

Comment:
Should these fields be added to the LinkML model, or is there a reason to exclude them?  If so, we'd need a new LinkML release and to update the @AGRCurationSchemaVersion annotation and the LinkMLSchemaConstants.java class.

Reply:
Ah, right. I can add these to the model.

Reply:
I've created a new LinkML release to include your changes (v2.6.1) so the `@AGRCurationSchemaVersion` annotation on this class needs to be updated and `LinkMLSchemaConstants.LATEST_RELEASE` needs to be bumped to 2.6.1